src/test/resources/api/products/_get_product_by_id/rs.json

```
@@ -0,0 +1,19 @@
+{
+  "id": 1,
+  "title": "iPhone 9",
+  "description": "An apple mobile which is nothing like apple",
+  "price": 549,
+  "discountPercentage": 12.96,
+  "rating": 4.69,
+  "stock": 94,
+  "brand": "Apple",
+  "category": "smartphones",
+  "thumbnail": "https://cdn.dummyjson.com/product-images/1/thumbnail.jpg",
+  "images": [
+    "https://cdn.dummyjson.com/product-images/1/1.jpg",
+    "https://cdn.dummyjson.com/product-images/1/2.jpg",
+    "https://cdn.dummyjson.com/product-images/1/3.jpg",
+    "https://cdn.dummyjson.com/product-images/1/4.jpg",
+    "https://cdn.dummyjson.com/product-images/1/thumbnail.jpg"
+  ]
+}
```

hardcoded values are not good here because if you'll try to get product by some other id rather than 1 your test will fail

---

done